content/en/examples/application/job/indexed-job.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: 'indexed-job'
+spec:
+  completions: 5
+  parallelism: 3
+  completionMode: Indexed
+  template:
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: 'worker'
+        image: 'gcr.io/<project>/job-worker'
```

Comment:
This might not be in quite line with our content guide:
https://kubernetes.io/docs/contribute/style/content-guide/#third-party-content

Maybe leave the `image` line commented out and tell readers that they must edit it for this to work.